lib/id3c/cli/command/etl/redcap_det.py

```diff
@@ -116,11 +116,11 @@ def decorated(*args, db: DatabaseSession, log_output: bool, **kwargs):
                             if not is_complete(instrument, redcap_record)
                     }
 
-                    if incomplete_instruments:
-                        LOG.debug(f"The following required instruments «{incomplete_instruments}» are not yet marked complete. " + \
-                                  f"Skipping REDCap DET {det.id}")
-                        mark_skipped(db, det.id, etl_id)
-                        continue
+                    # if incomplete_instruments:
+                    #     LOG.debug(f"The following required instruments «{incomplete_instruments}» are not yet marked complete. " + \
+                    #               f"Skipping REDCap DET {det.id}")
+                    #     mark_skipped(db, det.id, etl_id)
+                    #     continue
```

Comment:
I think this snuck into your commit while you were testing?

Reply:
yeah, looks like it. thanks